### PR TITLE
New blog post about Che deployment on AKS 

### PR DESCRIPTION
**Pull request by karatkep**
Our story about how we installed Eclipse Che on Microsoft Azure, what problems we met and how solved them.

### CODE REVIEW

**Comment by karatkep** [comment]
CC @l0rd to help with review

**Review by sparkoo — APPROVED** [review]
lgtm

**Comment by karatkep** [comment]
We added more details to the blog post. So this is a final version of proposed blog post.

@l0rd , @sparkoo , could you please review one more time? Thanks in advance!

**Review by ibuziuk — APPROVED** [review]
Thank you for the contribution, great content :+1: 
Added the suggestion for the title to make it more straightforward, drop the negative connotation (not work) and the particular versioning detail that is explained in the blog

**Comment by ibuziuk** [comment]
@karatkep could you please update the blog file with the publication date (we could probably target Monday 07.25) and make the file name match the blog's title. Will wait for the final approval from @l0rd next week before merging.

**Comment by karatkep** [comment]
@ibuziuk thank you. Updated according to the recommendations.

File: _posts/2022-07-25-installing-eclipse-che-on-aks.adoc [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+image::/assets/img/installing-eclipse-che-on-aks/aks-aad-server-app.png[Azure Kubernetes Service AAD Server]
+
+If we leave default settings of OAuth2 Proxy, leveraged in Gateway, it uses standard set of OIDC scopes and for them AAD returns an `access_token` that can be used against Microsoft Graph, not AKS.
+
+In order to obtain a token suitable for AKS, additional scope should be configured for OAuth2 Proxy, representing Azure Kubernetes Service AAD Server application:
+[source,yaml]
+----
+6dae42f8-4368-4678-94ff-3960e28e3630/user.read
+----
+where GUID is a Application ID for AKS registered in AAD.
```
Comment (l0rd) [inline _posts/2022-07-25-installing-eclipse-che-on-aks.adoc:74]:
```suggestion
where the ID is the Application ID of the AKS instance registered in AAD.
```

File: _posts/2022-07-25-installing-eclipse-che-on-aks.adoc [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+
+.Registered Che application
+image::/assets/img/installing-eclipse-che-on-aks/azure-che-demo-app.png[Registered Che application]
+
+WARNING: Don't forget to configure API permissions to authorize your application to call AKS Server API.
+
+.AKS API permissions
+image::/assets/img/installing-eclipse-che-on-aks/aks-api-permissions.png[AKS API permissions]
+
+After all preparations are completed, `chectl` deploy command can be used to install {prod} on AKS using yaml file above:
```
Comment (l0rd) [inline _posts/2022-07-25-installing-eclipse-che-on-aks.adoc:183]:
```suggestion
After the {prod} App configuration in Azure is completed, the command `chectl server:deploy` can be used to install {prod} on AKS using the `YAML` file above:
```

File: _posts/2022-07-25-installing-eclipse-che-on-aks.adoc [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+chectl server:deploy \
+       --platform=k8s \
+       --installer=operator \
+       --che-operator-cr-patch-yaml=che.yaml \
+       --skip-oidc-provider-check \
+       --skip-cert-manager \
+       --domain=eclipse-che-demo.mydomain.com
+----
+
+NOTE: In our case we already configured `cert-manager` and created `domain` according to the link:https://www.eclipse.org/che/docs/che-7/installation-guide/installing-che-on-microsoft-azure/[instruction].
```
Comment (l0rd) [inline _posts/2022-07-25-installing-eclipse-che-on-aks.adoc:195]:
```suggestion
NOTE: In our case we already configured `cert-manager` and created `domain` according to the link:https://www.eclipse.org/che/docs/che-7/installation-guide/installing-che-on-microsoft-azure/[old (7.41) instructions].
```

**Review by l0rd — COMMENTED** [review]
Thank you @karatkep, this blog post is great. There is a lot of work behind that. I made a few wording tweaks, please go through them and accept or comment.

**Comment by karatkep** [comment]
Thank you @l0rd. Proposed wording tweaks look perfect!

**Review by l0rd — APPROVED** [review]
LGTM

**Comment by l0rd** [comment]
Thank you @karatkep, feel free to [retweet](https://twitter.com/eclipse_che/status/1552008552293736450).

image

**Pull request merged**